### PR TITLE
Submission for this week

### PR DESCRIPTION
**Pull request by rckeely2**
Apologies, wrote it about two weeks ago but never committed...

### CODE REVIEW

File: exercises.ipynb [hunk 1]
```diff
@@ -65,7 +62,12 @@
     "    # Figure out how to select the text of the title\n",
     "    # of each legoset. A title should look like: \n",
     "    # \"10252: Volkswagen Beetle\"\n",
-    "    pass\n",
+    "    sections = soup.find('section', attrs={'class':'setlist'})\n",
+    "    title_list = []\n",
+    "    for article in sections.find_all('article', attrs={'class':'set'}):\n",
+    "        div = article.find('div', attrs={'class':'meta'})\n",
+    "        title_list.append(div.find(\"h1\").get_text())\n",
```
Comment (nandanrao) [inline exercises.ipynb:69]: What's wrong with css notation? `'article.set div.meta h1'`, all in one line?

File: exercises.ipynb [hunk 2]
```diff
@@ -150,22 +152,912 @@
...
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def iterate_page(url, first=False):\n",
+    "    request = requests.get(url)\n",
+    "    soup = BeautifulSoup(request.text, 'html.parser')\n",
+    "    page_div = soup.find('div', attrs={'class':'pagination'})\n",
+    "    last_url = page_div.find('li', attrs={'class':'last'}).find('a')[\"href\"]\n",
+    "    next_url = page_div.find('li', attrs={'class':'next'}).find('a')[\"href\"]\n",
```
Comment (nandanrao) [inline exercises.ipynb:167]: again: `'div.pagination li.last a'`

**Review by nandanrao — COMMENTED** [review]
Great!

**Comment by rckeely2** [comment]
I wrote it before you did the tutorial and wasn't aware of your much
shorter version...

**Pull request closed without merging**